Fix RPATH handling.
EasyBuild sets CMAKE_PREFIX_PATH to a ;-separated list of paths, so the original code is wrong here.

Maxime, 2024-01-09
diff -ru clr-rocm-5.6.0.orig/hipamd/CMakeLists.txt clr-rocm-5.6.0/hipamd/CMakeLists.txt
--- clr-rocm-5.6.0.orig/hipamd/CMakeLists.txt	2023-06-13 21:19:07.000000000 +0200
+++ clr-rocm-5.6.0/hipamd/CMakeLists.txt	2024-01-09 16:17:00.239317298 +0100
@@ -41,7 +41,7 @@
 
 # required to add the right link to libhsa-runtime in install/lib path
 # CMAKE_PREFIX_PATH is used as rpath to search for libs outside HIP
-set(CMAKE_INSTALL_RPATH "${CMAKE_PREFIX_PATH}/${CMAKE_INSTALL_LIBDIR}")
+set(CMAKE_INSTALL_RPATH "${CMAKE_PREFIX_PATH}")
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
 #############################
